File: 프로그래머스/lv2/12909. 올바른 괄호/올바른 괄호.py
def solution(s):
    answer = True
    count = 0
    for i in range(len(s)):
        if s[0] == ')' or count < 0:
            answer = False
            break
        if s[i] == '(':
            count += 1
        if s[i] == ')':
            count -= 1
    if count != 0:
        answer = False
    return answer

File: 프로그래머스/lv2/12909. 올바른 괄호/올바른 괄호.js
function solution(s){
    var answer = true;
    let count = 0;
    for(let i = 0; i < s.length; i++){
        if(s[0] === ')' || count < 0){
            answer = false;
            break;
        } else if (s[i] === '('){
            count++;
        } else {
            count--;
        }
    }
    if(count !== 0){
        answer = false;
    }

    return answer;
}